cypress/integration/profile/change_password.spec.js

```diff
@@ -0,0 +1,46 @@
+import existingUser from '../../../test-utils/mocks/existingUser';
+import { validationErrorMessages } from '../../../common/constants/messages';
+
+describe(`profile/change_password (unauthorized)`, () => {
+  it(`should redirect to login if not authorized`, () => {
+    // assert that route can't be reached without being authorized
+    cy.visit('/profile/change_password');
+    cy.get('nav[data-testid="Desktop Nav"]').should('exist');
+    cy.url().should('contain', '/login');
+    cy.url().should('not.contain', '/profile/update');
+    cy.get('h1').should('have.text', 'Login'); // redirect confirmed
+  });
+});
+
+describe('change_password', () => {
+  beforeEach(() => {
+    cy.clearCookies();
+
+    cy.server();
+    cy.route('POST', 'auth/password/change/').as('postChange');
+
+    cy.login();
+    cy.visit('/profile/change_password');
+    cy.get('h1').should('have.text', 'Enter new password');
+  });
+
+  it('should be able to change with valid passwords', () => {
+    cy.get('#newPassword1').type(existingUser.password);
+    cy.get('#newPassword2').type(existingUser.password);
+
+    cy.get('button[type="submit"]').click();
+
+    cy.wait('@postChange');
+
+    cy.contains(/You password has been changed to the new password/i);
+    cy.get('a[href="/profile"]').should('contain', 'Click here to return to Profile');
+  });
+
+  it('should NOT be able to change with mis-matched passwords', () => {
+    cy.get('#newPassword1').type(existingUser.password);
+    cy.get('#newPassword2').type(`${existingUser.password}1`);
+    cy.get('button[type="submit"]').click();
+
+    cy.get('div[role="alert"]').should('contain', validationErrorMessages.passwordMatch);
+  });
```

Comment:
What happens if the server is down?

Reply:
The standard `Something is wrong on our end. Please try again later.` message.  Want me to add a test for it?

Reply:
Yeah. Sorry that’s what I was getting at